source/user-portal/assets/script.js

```diff
@@ -12,45 +10,61 @@ window.addEventListener('DOMContentLoaded', init);
 
 async function init() {
 
+    
     let fetched = await fetchRecipes();
 
     if(!fetched){
         console.error('Failed fetch');
         return;
     }
 
+    collectStorage();
+
     createRecipeCards();
 }
 
+// Function here if I need to fetch the recipe... 
+// fetch(`https://api.spoonacular.com/recipes/${recipes[i]}/information?apiKey=`+apiKey)
 async function fetchRecipes() {
     return new Promise((resolve, reject) => {
 
         for(let i = 0; i < recipes.length; i++){
             fetch(recipes[i])
-            .then((response) => response.json())
-            .then((data) => {
-                recipeData[i] = data;
-                if(i + 1 == recipes.length){
-                    resolve(true);
-                }
+                .then(response => response.json())
+                .then(data =>{           
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8436327)
```suggestion
                .then(data => {           
```